core/src/main/java/io/grpc/internal/RetriableStream.java

```diff
@@ -195,7 +195,10 @@ public void run() {
             }
           }
           if (retryFuture != null) {
-            retryFuture.cancel(false);
+            boolean cancelled = retryFuture.cancel(false);
```

Comment:
> **Returns:**
> false if the task could not be cancelled, typically because it has already completed normally; true otherwise
> — https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/Future.html#cancel-boolean-

Do we need to distinguish between the future wasn't cancelled because it has already completed normally, and other unsuccessful cancel attempts?